src/fides/api/ops/service/connectors/consent_email_connector.py

```diff
@@ -97,20 +135,48 @@ def test_connection(self) -> Optional[ConnectionTestStatus]:
     def needs_email(
         self, user_identities: Dict[str, Any], privacy_request: PrivacyRequest
     ) -> bool:
-        """Schedules a consent email for consent privacy requests containing consent preferences and valid user identities"""
-
-        if not privacy_request.consent_preferences:
+        """Schedules a consent email for consent privacy requests containing consent preferences (old workflow) / privacy preferences
+        (new workflow) and valid user identities
+        """
+        if not privacy_request.policy.get_rules_for_action(
+            action_type=ActionType.consent
+        ):
             return False
 
-        consent_rules: List[Rule] = privacy_request.policy.get_rules_for_action(
-            action_type=ActionType.consent
+        old_workflow_consent_preferences: Optional[
+            Any
+        ] = privacy_request.consent_preferences
+        new_workflow_consent_preferences: List[
+            PrivacyPreferenceHistory
+        ] = filter_privacy_preferences_for_propagation(
+            self.configuration.system, privacy_request.privacy_preferences
         )
-        return bool(
-            consent_rules
-            and filter_user_identities_for_connector(self.config, user_identities)
+        if not (old_workflow_consent_preferences or new_workflow_consent_preferences):
+            return False
+
+        if not filter_user_identities_for_connector(self.config, user_identities):
+            return False
+
+        return True
+
+    def add_skipped_log(self, db: Session, privacy_request: PrivacyRequest) -> None:
+        """Add skipped log for the connector to the privacy request"""
+        ExecutionLog.create(
+            db=db,
+            data={
+                "connection_key": self.configuration.key,
+                "dataset_name": self.configuration.name,
+                "collection_name": self.configuration.name,
+                "privacy_request_id": privacy_request.id,
+                "action_type": ActionType.consent,
+                "status": ExecutionLogStatus.skipped,
+                "message": f"Consent email skipped for '{self.configuration.name}'",
+            },
         )
 
     def batch_email_send(self, privacy_requests: Query) -> None:
```

Comment:
batch_email_send is run once weekly and takes all consent privacy requests in a state of `awaiting_email_send` and batches relevant preferences together from all the privacy requests and sends in one email.

This also filters relevant preferences for this connector with `filter_privacy_preferences_for_propagation`.  Unlike the saas connector, if there are conflicting preferences, I send them both here.